Example/openxcframework/RecordingSourceController.swift

```diff
@@ -11,45 +11,46 @@ import openXCiOSFramework
 
 class RecordingSourceController: UIViewController,UITextFieldDelegate {
     
-    @IBOutlet weak var recswitch: UISwitch!
-    @IBOutlet weak var recname: UITextField!
-    @IBOutlet weak var dweetswitch: UISwitch!
-    @IBOutlet weak var dweetname: UITextField!
-    @IBOutlet weak var dweetnamelabel: UILabel!
-    @IBOutlet weak var uploadtraceswitch: UISwitch!
-    @IBOutlet weak var traceURLname: UITextField!
-    @IBOutlet weak var tergetURLnamelabel: UILabel!
-    @IBOutlet weak var apiSourcenamelabel: UILabel!
+    @IBOutlet weak var recordingSwitch: UISwitch!
+    @IBOutlet weak var recordingNameField: UITextField!
+    @IBOutlet weak var dweetSwitch: UISwitch!
+    @IBOutlet weak var dweetNameField: UITextField!
+    @IBOutlet weak var dweetNameLabel: UILabel!
+    @IBOutlet weak var uploadTraceSwitch: UISwitch!
+    @IBOutlet weak var traceUrlNameField: UITextField!
+    @IBOutlet weak var tergetUrlNameLabel: UILabel!
```

Comment:
As above, The variable name should start with "target", not "terget"

Reply:
done